content/docs/state-and-lifecycle.md

```diff
@@ -246,24 +246,24 @@ The `componentDidMount()` method runs after the component output has been render
   }
 ```
 
-Note how we save the timer ID right on `this`.
+Notez que l'on a enregistré l'ID du minuteur directement sur `this`.
 
-While `this.props` is set up by React itself and `this.state` has a special meaning, you are free to add additional fields to the class manually if you need to store something that doesn’t participate in the data flow (like a timer ID).
+Alors que `this.props` est mise en place par React lui-même et que `this.state` a un sens bien spécial, vous pouvez très bien ajouter manuellement d'autres champs sur la classe si vous avez besoin de stocker quelque chose qui ne participe pas au flux de données (comme un ID de minuteur).
```

Comment:
```suggestion
Alors que `this.props` est mis en place par React lui-même et que `this.state` a un sens bien spécial, vous pouvez très bien ajouter manuellement d'autres champs sur la classe si vous avez besoin de stocker quelque chose qui ne participe pas au flux de données (comme un ID de minuteur).
```

Micro-détail : puisqu'on parle de « champs », on va considérer `this.props` comme tel et accorder « mis » au masculin.

Reply:
D'accord, j'aurai plutôt dit que c'est une propriété de l'objet.